src/internal/__tests__/serverRenderer.test.js

```diff
@@ -0,0 +1,121 @@
+import React from "react"
+import styled, {
+  __DO_NOT_USE_OR_YOU_WILL_BE_HAUNTED_BY_SPOOKY_GHOSTS,
```

Comment:
Oh god 😂

Reply:
lol i know... this is where we run into that issue around `Invalid __context__ in force`, but styled-components 4 reworks the context so that error should go away.